workers/unity/Assets/Playground/Editor/SnapshotGenerator/SnapshotGenerator.cs

```diff
@@ -36,12 +36,6 @@ public struct Arguments
                 new WorkerAttributeSet(new Improbable.Collections.List<string> { UnityGameLogicType })
             });
 
-        private static readonly WorkerRequirementSet ClientSet = new WorkerRequirementSet(
```

Comment:
Drive-by warning fix.

Reply:
Was this just because it wasn't being used?

Reply:
Yep!